### PR TITLE
Refactor services.xml

### PR DESCRIPTION
**Pull request by msilvagarcia**
## Summary

Start separating XML configuration in multiple files to make the file easier to read.

## Tested scenarios

* Installing the plugin from scratch
* Updating the plugin from `develop`

### CODE REVIEW

**Review by cyattilakiss — DISMISSED** [review]

File: src/Resources/config/services/notification-processing.xml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+            <argument type="service" id="Adyen\Shopware\Service\ConfigurationService"/>
+            <argument type="service" id="Shopware\Core\Content\Media\MediaService"/>
+            <argument type="service" id="payment_method.repository"/>
+            <argument type="service" id="media.repository"/>
+            <argument>%shopware.media.enable_url_upload_feature%</argument>
+            <tag name="messenger.message_handler"/>
+            <call method="setLogger">
+                <argument key="$logger" type="service" id="monolog.logger.adyen_cron"/>
+            </call>
+        </service>
```
Comment (acampos1916) [inline src/Resources/config/services/notification-processing.xml:85]: Maybe the logo fetching services and cron belong in a different XML?

Comment (peterojo) [reply src/Resources/config/services/notification-processing.xml:85]: My preference would have been to have this file dedicated to all scheduled tasks (scheduled-tasks.xml), and move the notification receiver to a more generic services xml

File: src/Resources/config/services.xml [hunk 2]
```diff
@@ -6,236 +6,34 @@
...
-            <argument type="service" id="Adyen\Shopware\Service\ConfigurationService"/>
-            <argument type="service" id="Shopware\Core\Content\Media\MediaService"/>
-            <argument type="service" id="payment_method.repository"/>
-            <argument type="service" id="media.repository"/>
-            <argument>%shopware.media.enable_url_upload_feature%</argument>
-            <tag name="messenger.message_handler"/>
-            <call method="setLogger">
-                <argument key="$logger" type="service" id="monolog.logger.adyen_cron"/>
-            </call>
-        </service>
```
Comment (peterojo) [inline src/Resources/config/services.xml:238]: maybe we should also have a commands.xml file, we expect some more commands to pop up in the future

**Review by peterojo — APPROVED** [review]

File: src/Resources/config/services/notification-processing.xml (hunk 1 above)
Comment (acampos1916) [reply src/Resources/config/services/notification-processing.xml:85]: What do you think @msilvagarcia?

**Pull request merged**